src/platform/profile/avatar.ts

```diff
@@ -238,7 +238,7 @@ export namespace Avatar {
       },
       hasClaimedName: {
         type: 'boolean',
-        nullable: true
+        nullable: false
```

Comment:
I'd rather remove this line and add it to the `required` field as the rest of the mandatory ones